kedro/config/omegaconf_config.py

```diff
@@ -302,6 +308,13 @@ def _is_valid_config_path(self, path):
             ".json",
         ]
 
+    @staticmethod
+    def _register_new_resolvers(resolvers: dict[str, Callable]):
+        """Register custom resolvers"""
+        for name, resolver in resolvers.items():
+            if not OmegaConf.has_resolver("name"):
+                OmegaConf.register_new_resolver(name=name, resolver=resolver)
```

Comment:
Do you still think it makes sense to put a debug log here?

Reply:
I added it now!